src/components/forms/OfficeHoursForm.tsx

```diff
@@ -25,24 +25,27 @@ import { api } from './api';
 
 import { chakraStyles } from './selectStyles';
 import {
+  ADVICE,
   HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS,
+  INDIVIDUAL,
   PROJECT_CATEGORY_OPTIONS,
   TEAM,
   PROJECT_FEEDBACK,
-  TIMEZONE_OPTIONS,
```

Comment:
I notice my autoformatter removed trailing comma here.

Looks like that is the setting we configured:
https://github.com/ethereum/esp-website/blob/master/.prettierrc#L3

Not sure why this was here - @nhsz does your editor have different settings?

Reply:
Not my editor, I use the configured settings

Reply:
Ah ya @wackerow I see it was your commit: https://github.com/ethereum/esp-website/commit/ffdaa3e381af2e063bf44ca15a68645242d809fc

Please make sure you're using the config in the repo 😄